fe/fe-core/src/main/java/com/starrocks/mysql/MysqlHandshakePacket.java

```diff
@@ -34,7 +36,11 @@
     private static final MysqlCapability CAPABILITY = MysqlCapability.DEFAULT_CAPABILITY;
     // status flags not supported in StarRocks
     private static final int STATUS_FLAGS = 0;
-    private static final String AUTH_PLUGIN_NAME = "mysql_native_password";
+    private static final String DEFAULT_AUTH_PLUGIN_NAME = "mysql_native_password";
```

Comment:
Why to define two plugin name?

Reply:
change

Reply:
I think NATVIE_AUTH_PLUGIN_NAME is better

Reply:
DONE